tests/unit/hashexpire.tcl

```diff
@@ -2071,8 +2105,100 @@ start_server {tags {"hashexpire external:skip"}} {
             assert_keyevent_pattern $rd_replica hexpired myhash
 
             $rd_replica close
-        }
+            # Re-enable active expiry
+            $primary DEBUG SET-ACTIVE-EXPIRE yes
+            $replica_1 DEBUG SET-ACTIVE-EXPIRE yes
+        } {OK} {needs:debug}
         
+
+        test {Promotion to primary} {
```

Comment:
in what does this test differ from: test {Replica Failover/Promotion to Primary}?